src/main/java/com/hyperwallet/clientsdk/util/Request.java

```diff
@@ -0,0 +1,436 @@
+package com.hyperwallet.clientsdk.util;
+
+import cc.protea.util.http.Response;
+import cc.protea.util.http.BinaryResponse;
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.Proxy;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/*
+
+ Modified with modifications copyright Richard Stanford
+
+ -- Original portions by Joe Ernst --
+
+ * Copyright 2012 Joe J. Ernst
+ * <p/>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This class represents an HTTP Request message.
+ */
+public class Request extends Message<Request> {
```

Comment:
this code too please make sure we are not legally bounded on this, thanks

Reply:
<img width="1014" alt="Screen Shot 2022-02-15 at 9 52 31 AM" src="https://user-images.githubusercontent.com/91630355/154120479-26d67f23-d0b1-46dc-a026-5c844ae4d49a.png">

We should be covered for this